plugins/vpp/ifplugin/vppcalls/nat_vppcalls.go

```diff
@@ -31,6 +32,13 @@ const (
 // NoInterface is sw-if-idx which means 'no interface'
 const NoInterface uint32 = 0xffffffff
 
+// Default NAT virtual reassembly values
+const (
+	MaxReassembly = 1024
+	MaxFragments  = 5
+	Timeout       = 2
```

Comment:
These are not very informative names in package that contains various configuration for NAT.
It will look like: `vppcalls.Timeout`. I would actually propose to move it from vppcalls entirely since it's used in nat configurator only.